pkg/config/service_config.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import "github.com/spf13/pflag"
+
+const (
+	flagLoadBalancerClass    = "load-balancer-class"
+	defaultLoadBalancerClass = "elbv2.k8s.aws"
```

Comment:
This definitely needs to change to be a prefixed name, since ["Unprefixed names are reserved for end-users."](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class).

Reply:
updated